low-level/framework/utils/utility.py

```diff
@@ -80,6 +83,54 @@ def get_os(self):
             else:
                 return None
 
+    def get_epoch_time(self, date, _time):
```

Comment:
Also address these codacy warnings, for new code atleast we should be targeting
https://docs.quantifiedcode.com/python-anti-patterns/correctness/method_could_be_a_function.html

Reply:
'Method could be a function' comes for the ambiguous functions, which does not refer to any instance or class.
Here these functions are using self keyword.
Not sure why still this warning coming.
Except this warning, addressed other codacy warnings